packages/client/src/views/IssueCertificate/IssueCollectorForm/IssueCollectorForm.test.tsx

```diff
@@ -0,0 +1,465 @@
+  id: '6a5fd35d-01ec-4c37-976e-e055107a74a1',
+  data: mockDeclarationData,
+  event: Event.Birth
+}
+
+beforeEach(() => {
+  const s = createStore()
+  store = s.store
+  history = s.history
+  location = createLocation('/')
+  history.location = location
+})
+
+describe('Certificate issue collector test for a birth registration without father details', () => {
+  describe('Test collector group', () => {
+    let component: ReactWrapper<{}, {}>
+
+    beforeEach(async () => {
+      const testComponent = await createTestComponent(
+        <IssueCollectorForm
+          location={location}
+          history={history}
+          //@ts-ignore
+          declaration={birthDeclarationForIssuance}
+          match={{
+            params: {
+              registrationId: '6a5fd35d-01ec-4c37-976e-e055107a74a1',
+              eventType: 'birth',
+              groupId: 'collector'
+            },
+            isExact: true,
+            path: '',
+            url: ''
+          }}
+        />,
+        { history, store }
+      )
+      component = testComponent
+    })
+
+    it('father option will be available', async () => {
+      await waitForElement(component, '#type_FATHER')
+      expect(component.find('#type_FATHER').hostNodes()).toHaveLength(1)
+    })
+
+    it('continue button disabled when not selected', async () => {
+      await waitForElement(component, '#continue-button')
+      expect(
+        component.find('#continue-button').hostNodes().props().disabled
+      ).toBeTruthy()
+    })
+
+    describe('Test the component with certificate', () => {
+      beforeEach(async () => {
+        const updatedCertificateArray = [
+          {
+            collector: {
+              type: 'FATHER'
+            }
+          }
+        ]
+
+        const updatedMockDeclarationData = {
+          ...birthDeclarationForIssuance,
+          data: {
+            ...birthDeclarationForIssuance.data,
+            registration: {
+              ...birthDeclarationForIssuance.data.registration,
+              certificates: updatedCertificateArray
+            }
+          }
+        }
+
+        const testComponent = await createTestComponent(
+          <IssueCollectorForm
+            location={location}
+            history={history}
+            //@ts-ignore
+            declaration={updatedMockDeclarationData}
+            match={{
+              params: {
+                registrationId: '6a5fd35d-01ec-4c37-976e-e055107a74a1',
+                groupId: 'collector'
+              },
+              isExact: true,
+              path: '',
+              url: ''
+            }}
+          />,
+          { history, store }
+        )
+        component = testComponent
+      })
+
+      it('redirects to id check component upon FATHER option selection', async () => {
+        component
+          .find('#type_FATHER')
+          .hostNodes()
+          .simulate('change', { target: { value: 'FATHER' } })
+
+        component.update()
```

Comment:
Do we need to select the father option? Are we not already providing the component with the value in the declaration?